src/com/example/module03/PassportFormCreator_Task04.java

```diff
@@ -0,0 +1,17 @@
+package com.example.module03;
+
+import java.util.Scanner;
+
+public class PassportFormCreator_Task04 {
+    public void createForms() {
+        Scanner scanner = new Scanner(System.in);
+        String name = scanner.nextLine();
+        String surname = scanner.nextLine();
+        int years = scanner.nextInt();
+        System.out.println(name + " " + surname + " " + (years *= 10));
```

Comment:
it is not obvious to change value of years inside of sout. introduce a new variable and give it appropriate name according the task. use this new variable for output value where it needed.